backend/graminsta/post/serializers.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: UTF-8 -*-
+"""serializers.py"""
+
+from django.contrib.auth import models as auth_models
+from rest_framework import serializers
+from .models import FollowRelationship
+
+
+class FollowSerializer(serializers.ModelSerializer):
+    """
+    Serializer that serializes/deserializes
+    FollowRelationship object
+    """
+    class Meta:
+        """
+        Meta Information
+        """
+        model = FollowRelationship
+        fields = (
+            'from_user', 'to_user'
+        )
+
+
+class UserSerializer(serializers.ModelSerializer):
```

Comment:
Can you reuse `UserSerializer` in `core` instead of introducing more.

Reply:
I'm a little not sure which would be better, reuse `UserSerializer` in `core`, or redefine the `UserSerializer` in post app so that I can make sure its fields read_only and only contains PUBLIC properties? For now, I just reuse the code.